app/scripts/editor/app.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+require('angular');
+
+angular.module('Liberator.new_article', [
+    require('angular-ui-router'),
+])
+   .config(require('./config'))
+   .controller('newArticleController', require('./controller.js'))
```

Comment:
By convention introduced by Angular core team, controller names should be `PascalCase`